### ISSUE TITLE
Style adjustment for disabled fields

### ISSUE THREAD

**Issue by waitingallday** [unimelb/unimelb-design-system#676]
#### Expected behaviour

Make it visibly clear that the field is not editable, by removing the input box style on disabled controls.
#### Actual behaviour

Currently uses the browser default (no focus style). We haven't provided any guidance to developers for how a disabled form control should look!
#### Steps to reproduce

N/A

Proposed style:

![screen shot 2016-07-25 at 4 46 10 pm](https://cloud.githubusercontent.com/assets/863542/17092677/7c5d0258-5287-11e6-9d52-e43dde70fdc7.png)

**Comment by axelboc** [comment]
Playing on the opacity might be enough, and can easily be used on other form controls for consistency. It still needs to look like a text field, I reckon. Perhaps also removing the box shadow and setting the `not-allowed` cursor?

<img width="478" alt="screen shot 2016-07-26 at 8 33 32 am" src="https://cloud.githubusercontent.com/assets/2936402/17119891/2cdce646-530c-11e6-9131-6e8478564b38.png">

**Comment by waitingallday** [comment]
I was talking to @anormand about this, and one of the most common problems he's seen with the web apps he audits is the colour contrast on disabled fields. I like the idea of removing all ambiguity with a disabled field - that it just looks like text, so there is no way a user thinks that it might be editable, but happy to refine the style a bit further.

**Comment by axelboc** [comment]
There was more discussion on this with @anormand, if I recall. In guideline [1.4.3 Contrast (minimum)](https://www.w3.org/TR/WCAG20/#visual-audio-contrast-contrast) on minimum contrast requirement, WCAG specifically excludes "inactive user interface components", so it's okay for disabled fields to not have sufficient colour contrast. Personally, I'd strongly advise against removing the border and background colour from disabled fields, as I'm pretty sure this would go against other WCAG guidelines, like predictability ([3.2.4 Consistent Identification](https://www.w3.org/TR/WCAG20/#consistent-behavior-consistent-functionality)).
